service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -516,7 +516,41 @@ Resources are configured under `spec.runtime.<component>`. The following compone
 |v2.0 - tech preview
 |===
 
-For an example, of how to configure Pilot resource scheduling see xref:../../service_mesh/v2x/ossm-custom-resources.adoc#ossm-cr-pilot_ossm-custom-resources-v2x[Istio Pilot configuration]
+To schedule resources and limit resource allocation, use the following example. 
+
+.Example Pilot parameters
```

Comment:
@rcernich I don't want to use Pilot for this example because moving forward. Would it be broadly similar if I changed components?

Reply:
Those settings should apply to all components.  Some components may not support all settings, but those cases should be considered bugs, so it's okay to include it for all components in the docs.  If there are any special cases, we could mention those individually.  We might want to add a `requests` block to resources as well (resources have two settings, requests and limits, but I forget the difference off the top of my head).  Here's a list of all the [component names](https://github.com/maistra/istio-operator/blob/maistra-2.1/pkg/apis/maistra/v2/runtime.go#L28-L69).  Some might not be applicable to all versions, and for the names with dots, the dots are part of the name (e.g. `tracing.jaeger.allInOne` applies to Jaeger's all-in-one pod/container, which wouldn't apply if you're not installing jaeger or not using all-in-one).